src/lib/y2firewall/widgets/overview.rb

```diff
@@ -51,6 +51,8 @@ class OverviewTreePager < CWM::TreePager
       def initialize
         textdomain "firewall"
 
+        @fw = Y2Firewall::Firewalld.instance
+        @fw.read # FIXME when?
```

Comment:
Probably I would read the configuration before running the dialog or in the initialization of the client:

https://github.com/yast/yast-firewall/pull/71/commits/71b918de1df14f322b803cd33227dfe1cfb9b1e7#diff-78381f0fec693f47b838ca5d925d0352R32